components/admin_console/admin_definition.jsx

```diff
@@ -2483,9 +2483,17 @@ export default {
                             onConfigLoad: (configVal) => configVal / MEBIBYTE,
                             onConfigSave: (displayVal) => displayVal * MEBIBYTE,
                         },
+                        {
+                            type: Constants.SettingsTypes.TYPE_BOOL,
+                            key: 'ImageProxySettings.Enable',
+                            label: t('admin.image.enableProxy'),
+                            label_default: 'Enable Image Proxy:',
+                            help_text: t('admin.image.enableProxyDescription'),
+                            help_text_default: 'When true, enables an image proxy for loading all Markdown images.',
+                        },
                         {
                             type: Constants.SettingsTypes.TYPE_DROPDOWN,
-                            key: 'ServiceSettings.ImageProxyType',
+                            key: 'ImageProxySettings.ImageProxyType',
```

Comment:
Can you remove the "None" value for the image proxy type? I don't think that one is valid any more, so it probably won't let you save if that's selected